scripts/save.sh

```diff
@@ -108,6 +109,13 @@ pane_full_command() {
 	$strategy_file "$pane_pid"
 }
 
+capture_pane_contents() {
+	local pane_id="$1"
+	local start_line="0"
+	[[ "$(get_tmux_option "$pane_contents_area_option" "visible")" == "full" ]] && start_line="-$2"
+	tmux capture-pane -ep -S "$start_line" -t "$pane_id" > "$(resurrect_pane_file "$pane_id")"
```

Comment:
I was just reading up about `capture-pane` flags, what do you think about adding `-J` option?
So, after restore, if a user resizes pane/window, the text should automatically re-wrap to the new width.